packages/loader/container-loader/src/container.ts

```diff
@@ -821,14 +822,16 @@ export class Container
 		const clientType = `${interactive ? "interactive" : "noninteractive"}${
 			type !== undefined && type !== "" ? `/${type}` : ""
 		}`;
+
+		this._containerId = uuid();
 		// Need to use the property getter for docId because for detached flow we don't have the docId initially.
 		// We assign the id later so property getter is used.
 		this.subLogger = createChildLogger({
 			logger: subLogger,
 			properties: {
 				all: {
 					clientType, // Differentiating summarizer container from main container
-					containerId: uuid(),
+					containerId: this._containerId,
```

Comment:
when we log this telemetry event,  will we have problems with the Data_containerId ?

Reply:
What do you mean by that? I am just initializing it above in a variable rather than just here as it was not used anywhere else before.

Reply:
please disregard, I missed the removal of the uuid :)